lib/wickedbase.pm

```diff
@@ -258,8 +292,8 @@ sub setup_tunnel {
     my $local_ip = $self->get_ip(type => 'host');
     my $remote_ip = $self->get_remote_ip(type => 'host');
     my $tunnel_ip = $self->get_ip(type => $type);
-    file_content_replace($config, local_ip => $local_ip, remote_ip => $remote_ip, tunnel_ip => $tunnel_ip);
-    $self->wicked_command('ifup', $type);
+    file_content_replace($config, local_ip => $local_ip, remote_ip => $remote_ip, tunnel_ip => $tunnel_ip, iface => iface());
+    $self->wicked_command('ifup', 'all');
     assert_script_run('ip a');
```

Comment:
isn't `wicked_command` make this line useless ?

Reply:
not really:

`my $cmd = '/usr/sbin/wicked --log-target syslog ' . $action . ' ' . $iface;`

Reply:
I meant this https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/wickedbase.pm#L46